app/src/main/java/com/simplecity/amp_library/ui/screens/nowplaying/PlayerFragment.java

```diff
@@ -328,11 +329,18 @@ public void onResume() {
 
             disposables.add(sharedSeekBarEvents.subscribe(
                     seekBarChangeEvent -> {
+                        UiEventType uiEventType = null;
                         if (seekBarChangeEvent instanceof SeekBarStartChangeEvent) {
+                            uiEventType = UiEventType.SEEK_START;
                             isSeeking = true;
                         } else if (seekBarChangeEvent instanceof SeekBarStopChangeEvent) {
+                            uiEventType = UiEventType.SEEK_STOP;
                             isSeeking = false;
                         }
+                        Song song = MusicServiceConnectionUtils.getSong();
+                        if (uiEventType != null)
+                            newUiEvent(song, uiEventType, getContext(), MusicServiceConnectionUtils
```

Comment:
Is the position supposed to come from `MusicServiceConnectionUtils.getPosition()` here?

In https://github.com/CUTR-at-USF/MUSER/pull/109#issuecomment-730020123 you said:

>Fetching the position from the seekBar's progress listener solved the problem.

Just wanted to confirm. And if so, please add a comment here about why `MusicServiceConnectionUtils.getPosition()` is called here rather getting it inside the method - in both cases `MusicServiceConnectionUtils.getPosition()` is called so looking at code it's not clear why `newUiEvent()` can't be simplified to remove the `seekPosition` parameter.

Reply:
Yes - the value of `seekPosition` is more accurate when calculated from here compared to when it is calculated from the `EventUitls.newUiEvent()`.